7a051bb/template/css/home-hero.css

```diff
@@ -675,13 +655,22 @@
 /* Disabling most of the Firefox animation on Android -- low browser share and too many bugs */
 
 @-moz-document url-prefix() {
+    .homeHero {
+        @container (max-width: 899px) {
+            &, & * {
+                animation: none;
+            }
+        }
```

Comment:
Forcing everything below to have no animation

ugh... this is so ugly. I'm sorry